adding new annotation to specify in which container inside pod backup should run (#820)
k8up-io/k8up: docs/modules/ROOT/pages/how-tos/application-aware-backups.adoc

2022-02-14T18:07:21.711+0100	79020 document(s) restored successfully. 0 document(s) failed to restore.
----
====

> == Pick specific container

Reviewer: ```suggestion
== Pick a specific container
```

> Using `k8up.io/backupcommand-container` annotation You can specify in which container inside pod backup command should be executed. 

Reviewer: ```suggestion
Using `k8up.io/backupcommand-container` annotation You can specify the container name inside the pod where the command will get executed. 
```
